gnovm/pkg/packages/download.go

```diff
@@ -0,0 +1,83 @@
+package packages
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/gnolang/gno/tm2/pkg/bft/rpc/client"
+	"github.com/gnolang/gno/tm2/pkg/errors"
+)
+
+func DownloadModule(pkgPath string, dst string) error {
+	modFilePath := filepath.Join(dst, ModfileName)
+	if _, err := os.Stat(modFilePath); os.IsNotExist(err) {
+		fmt.Fprintln(os.Stderr, "gno: downloading", pkgPath)
```

Comment:
we usually don't use os.StdXXX, but stdouterr from a struct (io something) which is passed when needed.

Reply:
done in [e648305](https://github.com/gnolang/gno/pull/2932/commits/e648305445aa2a10c5ed16357ef2da00560879df)